index.php

```diff
@@ -49,26 +53,35 @@
     </div>
 </header>
 
-<section id="items">
+<section id="list">
     <?php
-    $items = ["test"];
+
+
+    $query = $pdo->prepare('SELECT `shows`.`name`, `genres`.`name` AS `genre`, `release year`, `image`, `alt` FROM `shows` INNER JOIN `genres` ON `shows`.`genre` = `genres`.`id`;');
+
+    $query->execute();
+
+    $items = $query->fetchALL();
+
     $html = "";
     foreach ($items as $item) {
-        $html = '<div class="item1">'
-            . '<img src="" alt="">'
+        $html = '<div class="container">'
+            . '<div style="background-image: url(images/' . $item['image'] . ')" class="item1"></div>'
+            . '<p>' . $item['name'] . '</p>'
+            . '<p>' . $item['genre'] . '</p>'
+            . '<p>' . $item['release year'] . '</p>'
+            //. '<img class ="picture" src="images/' . $item['image'] . '" alt="' . $item['alt'] . '">'
```

Comment:
Can you remove these commented out bits?

Reply:
yeah

Reply:
fixed